tests/e2e/test_rosacli_oidc_config.go

```diff
@@ -249,9 +266,17 @@ var _ = Describe("Register ummanaged oidc config testing",
 			Expect(installerRole).ToNot(BeNil())
 			roleArn := installerRole.RoleArn
 
+			By("Create a temp dir to execute the create commands")
+			output, err := rosaClient.Runner.RunCMD([]string{"mktemp", "-d"})
+			Expect(err).To(BeNil())
+			dirToClean = strings.TrimSpace(output.String())
```

Comment:
A common function to create the temporary file could be appreciated.

Reply:
I will add the common function if other place needs that in future